src/react/src/components/ContributeForm.jsx

```diff
@@ -114,11 +115,25 @@ class ContributeForm extends Component {
             error && error.length ? (
                 <React.Fragment>
                     <ul>
-                        {error.map(err => (
-                            <li key={err} style={{ color: 'red' }}>
-                                {err}
-                            </li>
-                        ))}
+                        {error.map(err => {
+                            const parsedString = parser
+                                .parseFromString(err, 'text/html')
+                                .querySelector('body').innerHTML;
+                            function createMarkup() {
```

Comment:
1. Can this be replaced into separate component?
2. This looks like anti-pattern, you try to get access to the DOM within render phase without using React refs.
3. You created `function createMarkup()` inside `.map` iteration which doesn't look clean in terms of performance.

Reply:
rewritten